src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -107,8 +114,9 @@ public void save(AddressBook addressBook) throws StorageOperationException {
      * Loads data from this storage file.
      *
      * @throws StorageOperationException if there were errors reading and/or converting data from file.
+     * @throws StorageFileDeletedException if the storage file is deleted when the program is running.
      */
-    public AddressBook load() throws StorageOperationException {
+    public AddressBook load() throws StorageOperationException, StorageFileDeletedException {
```

Comment:
probably don't need this. 
load doesn't need to check for this exception.

try to understand the context of the problem, this exception is intended to handle accidental file deletes during runtime.